rust/ql/src/queries/security/CWE-614/InsecureCookie.ql

```diff
@@ -74,7 +74,9 @@ module PartitionedCookieConfig implements DataFlow::ConfigSig {
     node instanceof Barrier
   }
 
-  predicate observeDiffInformedIncrementalMode() { any() }
+  predicate observeDiffInformedIncrementalMode() {
+    none() // only used negatively
+  }
```

Comment:
This appears to be a "drive by" change in the sense that it's not about the source and sink locations.  It looks correct, this configuration is only used to filter results.